src/handlers/comment/handlers/unassign.ts

```diff
@@ -0,0 +1,38 @@
+import { addAssignees, addCommentToIssue, removeAssignees } from "../../../helpers";
+import { getBotContext, getLogger } from "../../../bindings";
+import { Payload } from "../../../types";
+import { IssueCommentCommands } from "../commands";
+import { GLOBAL_STRINGS } from "../../../configs";
+
+export const unassign = async (body: string) => {
+  const { payload: _payload } = getBotContext();
+  const logger = getLogger();
+  if (body != IssueCommentCommands.UNASSIGN && body.replace(/`/g, "") != IssueCommentCommands.UNASSIGN) {
+    logger.info(`Skipping to unassign. body: ${body}`);
+    return;
+  }
+
+  const payload = _payload as Payload;
+  logger.info(`Received '/unassign' command from user: ${payload.sender.login}`);
+  const issue = (_payload as Payload).issue;
+  if (!issue) {
+    logger.info(`Skipping '/unassign' because of no issue instance`);
+    return;
+  }
+
+  const issue_number = issue!.number;
+  const _assignees = payload.issue?.assignees;
+  const assignees = _assignees ?? [];
+  let shouldUnassign = false;
+  assignees.forEach((assignee) => {
+    if (assignee === payload.sender.login) {
+      shouldUnassign = !shouldUnassign && true;
```

Comment:
I'm confused about this `&& true` syntax

Reply:
it's basically shouldUnassign = true

Reply:
We want 1 active assignee at a time so you can update it like 
`shouldUnassign = assignees.length > 0 && payload.sender.login === assignees[0]`